Formula/pnpm.rb

```diff
@@ -0,0 +1,19 @@
+class Pnpm < Formula
+  desc "📦🚀 Fast, disk space efficient package manager "
```

Comment:
```
Error: 1 problem in 1 formula detected
pnpm:
  * C: 2: col 54: Description shouldn't have a trailing space
```

Remove that trailing space and we should be good to go. 🎉

Reply:
Didn't notice that, thx.